### PR TITLE
JDG-2449 Quickstart build problems

### PR DESCRIPTION
**Pull request by gustavolira**

### CODE REVIEW

File: settings.xml [hunk 1]
```diff
@@ -1,112 +1,131 @@
...
+      </pluginRepositories>
+    </profile>
+
+    <profile>
+      <id>mead-jdg7</id>
+      <repositories>
+        <repository>
+          <id>mead-jdg7</id>
+          <name>MEAD JDG7 Maven Repository</name>
+          <url>http://download.lab.eng.bos.redhat.com/brewroot/repos/jb-dg-7-rhel-7-build/latest/maven/</url>
```
Comment (diegolovison) [inline settings.xml]: Should we have this repo upstream?

**Review by diegolovison — CHANGES_REQUESTED** [review]

File: settings.xml (hunk 1 above)
Comment (gustavolira) [reply settings.xml]: @diegolovison without that we receive a lot of errors on jenkins related with `org.infinispan:infinispan-bom` all the JDG-2449 errors comes from jenkins

Comment (gustavolira) [reply settings.xml]: @diegolovison I forced the branch updated and changed the commit removing the settings.xml changes

**Comment by gustavolira** [comment]
Closing in favor of #248

**Pull request closed without merging**